pyfixest/estimation/feiv_.py

```diff
@@ -460,7 +499,9 @@ def eff_F(self) -> None:
 
         # Extract the submatrix
 
```

Comment:
Could we not use the output of `iv_loc = self.extract_indices_IVs()` here as well?

Reply:
Thanks, I just fixed this part by replacing with one line of code!